build.xml

```diff
@@ -54,18 +54,19 @@
     <!-- FIXTURES -->
     <target name="fixtures" depends="doctrine">
         <echo message="Loading data fixtures..." />
-        <exec executable="bin/console" checkreturn="true">
-            <arg value="doctrine:fixtures:load" />
-            <arg value="--no-interaction" />
-        </exec>
+        <phingcall target="fixturesRun"/>
     </target>
     <target name="fixturesDev" depends="behat">
```

Comment:
Fixtures dev should load fixtures dev and fixtures test should load fixtures test.